### ISSUE TITLE
Black - Code Formatting

### ISSUE THREAD

**Issue by Zarathustra2** [cb109/payshare#15]
Maybe we should add formatting rules to this project for example: [Black](https://github.com/python/black)

**Comment by cb109** [comment]
I agree, especially since Django will also do this once black is stable (https://github.com/django/deps/blob/master/accepted/0008-black.rst). I am currently getting some experience with this, I think it would be nice to set this up as a https://pre-commit.com/ hook which has builtin support for black.
